packages/@uppy/angular/projects/uppy/angular/package.json

```diff
@@ -25,8 +25,8 @@
     "tslib": "^2.0.0"
   },
   "peerDependencies": {
-    "@angular/common": "^17.3.0",
-    "@angular/core": "^17.3.0",
+    "@angular/common": "^18.0.0",
+    "@angular/core": "^18.0.0",
```

Comment:
Why not allow both 17 and 18?

Reply:
https://github.com/transloadit/uppy/pull/5067#discussion_r1559140737

Reply:
I still don't agree with that. We either deprecate the plugin or we show the bare minimal effort in maintaining this, which includes at the minimum knowing at which versions it worked. If it worked on 17 before, and you tested it on 18, then we can support both. 

If it doesn't workout somehow, we can always remove 17x support as a bugfix.

Reply:
Is the upgrade tool made for consumers or for library authors? My guess is the former, meaning that we should double guess it for a library. It's common to support multiple versions of a framework if it comes for free. We do the same for Svelte and React. Vue 2 is EOL so that's a different story. Angular 18 _just_ came out, so the majority of people are still on 17 and will be stuck there for a long time potentially.

I think it's important we support both, just like our other integrations.

Reply:
```suggestion
    "@angular/common": "ˆ17.0.0 || ^18.0.0",
    "@angular/core": "^17.0.0 || ^18.0.0",
```